clients/fossid-webapp/src/main/kotlin/ReponseBody.kt

```diff
@@ -20,14 +20,14 @@
 package org.ossreviewtoolkit.clients.fossid
 
 open class EntityPostResponseBody<T> {
-    var operation: String? = null
```

Comment:
I am a little concerned by these changes: did you test them ?
How the JSON mapper will be able to set the values of these properties if they are immutable ?

Reply:
> I am a little concerned by these changes: did you test them ?

Well, `FossIdClientNewProjectTest.kt` passes. That test should have caught any issues, right? (As `EntityPostResponseBody<Any>.toList()` is called there.)

> (we are not in a constructor here)

Good question actually 😁 I believe some Jackson reflection magic is involved here... but actually, what hindern us to *make* these constructor parameters? I'll give that a try.

Reply:
Ok I crosschecked the data in `fossid-webapp/src/test/assets` and it seems we cover both cases, with and without error in the response.

> Good question actually 😁 I believe some Jackson reflection magic is involved here... 

Right, now I remember the Kotlin/Jackson library has some special trick to go around `val`.

Reply:
Should be solved now away, as I've changed the classes to have constructors.